### ISSUE TITLE
Provide native compressed files for OSX Kivy installer

### ISSUE THREAD

**Issue by ryandoherty** [kivy/kivy#3968]
It would make the OSX install process a bit easier if a compressed file format was used that OSX supports natively. Needing to install another app just to install Kivy is a pain and unnecessary.

**Comment by akshayaurora** [comment]
I guess we could provide both formats people could choose to download a format that works out of the box but is twice the size.

**Comment by ryandoherty** [comment]
Keka is 19MB, so I'm not sure the benefit of using .7z if you need to download Keka along with the Kivy .7z file too.

**Comment by kived** [comment]
Keka can be used for extracting other archives in the future, and isn't specific to Kivy. If another format would literally be twice the size, that would also be larger than the Kivy 7z + Keka, so 7z means you still have a smaller download overall.

That said, many OSX users are brainwashed to want everything done the Apple way, and I think the primary reason for reducing download size is to benefit people in places with limited or slow internet connections, because the package isn't really that big. And frankly, people in places with limited or slow connections probably can't afford Apple hardware.

**Comment by akshayaurora** [comment]
@kived nice bait!

**Comment by ryandoherty** [comment]
That's a lot of assumptions and a bit of a slam against OSX users, which isn't necessary. 

Not needing to install a separate piece of software makes the install process easier at the cost of a longer download time. As a Kivy user I would rather not need to install a separate piece of software to install it.

**Comment by akshayaurora** [comment]
@ryandoherty don't fret he is just trying to have some fun with me :) we will be providing a separate package shortly.

**Comment by ryandoherty** [comment]
Dammit! I need more coffee :)

**Comment by brentpicasso** [comment]
hugs his linux running on his samsung series 9 <3

**Comment by ijstokes** [comment]
+1 on please don't use 7z. What a hassle! Are you guys really that bandwidth starved?  A DMG will make life much easier for people.  Here is what I get in terms of file sizes:

```bash
(testdryrun)ijstokes@petawawa ~ $ du -hs ~/Downloads/Kivy2/Kivy2.app
236M	/Users/ijstokes/Downloads/Kivy2/Kivy2.app

(testdryrun)ijstokes@petawawa ~ $ ls -Flahd ~/Downloads/Kivy-1.9.1-osx-python2.7z 
-rw-r--r--@ 1 ijstokes  staff    46M May  6 10:38 /Users/ijstokes/Downloads/Kivy-1.9.1-osx-python2.7z

(testdryrun)ijstokes@petawawa ~ $ ls -Flahd ~/Downloads/Kivy2*
-rw-r--r--@ 1 ijstokes  staff    92M May  6 11:50 /Users/ijstokes/Downloads/Kivy2-TestDMG.dmg
```

So yes, 92MB using DMG vs. 46MB with 7-zip.

**Comment by kived** [comment]
@ijstokes Not all Kivy users have access to fast and unlimited connections - just because it would make **your** life easier doesn't mean it would make it easier for everyone. We will **always** provide a 7z archive, because it's very helpful for some users. We may decide to provide a DMG in addition, but 7z won't go away.

**Comment by RMKD** [comment]
ditto on the 7z is unfriendly to users comment. any plans for that dmg? (or a tar.gz?)

**Comment by dessant** [comment]
This will be fixed with the release of Kivy 1.10.0.
